.gitlab/source_test/linux.yml

```diff
@@ -70,25 +81,25 @@ tests_deb-x64-py2:
   rules:
     - !reference [.on_a6]
   variables:
-    PYTHON_RUNTIMES: '2'
+    PYTHON_RUNTIMES: "2"
```

Comment:
These cosmetics change make the PR needlessly longer than it should :(

They should be in a separated commit at the very least IMHO

Reply:
yes it was not intended, I can remove this it's useless